cli/src/cli_output.rs

```diff
@@ -839,3 +840,53 @@ impl From<&Lockout> for CliLockout {
         }
     }
 }
+
+#[derive(Serialize, Deserialize, Default)]
+pub struct CliSignOnlyCommand {
```

Comment:
Let's rename this slightly to describe the contents, not the caller. How about `CliSignOnlyData`?